src/structures/BaseCommandInteraction.js

```diff
@@ -123,6 +125,15 @@ class BaseCommandInteraction extends Interaction {
       }
     }
 
+    if (attachments) {
+      result.attachments = new Collection();
+      for (const attachment of Object.values(attachments)) {
+        const patched = new MessageAttachment();
+        patched._patch(attachment);
```

Comment:
Would this work?
```suggestion
        const patched = new MessageAttachment(attachment);
```

Reply:
@ImRodry No I don't think this would work as the first param the constructor for `MessageAttachment` takes is the `file` param which expects a `string` or a `Buffer`

Reply:
Can’t you obtain a buffer from the data you receive?